src/test/groovy/com/causecode/fileuploader/ufile/renewer/TemporaryUrlRenewerServiceSpec.groovy

```diff
@@ -19,8 +19,7 @@ import spock.lang.Specification
  * @author Milan Savaliya
  * @since 3.1.3
  */
-@TestFor(TemporaryUrlRenewerService)
-class TemporaryUrlRenewerServiceSpec extends Specification {
+class TemporaryUrlRenewerServiceSpec extends Specification implements ServiceUnitTest<TemporaryUrlRenewerService> {
```

Comment:
is implementing BuildDataTest not necessary?

Reply:
We are not creating any domain instance in this test file, so it's not needed here.